### DISCUSSION TITLE
Preventing tab unloader for active tabs (e.g. uploading files)

### DISCUSSION THREAD

**Discussion by redrumthebum** [zen-browser/desktop#4021] [Ideas]
i've had so many times where i forgot i was uploading something on a different tab and i had to redo the upload because it got unloaded 😅 an option like this might help with that. wdyt?

**Comment by EtaoinWu** [comment]
Also I suggest not to unload tabs that are playing video/audio or in calls (using camera, microphone or screen sharing).
